src/display/mapwindow.cpp

```diff
@@ -189,19 +191,29 @@ void MapWindow::slot_setScrollBars(const Coordinate &min, const Coordinate &max)
     updateScrollBars();
 }
 
+void MapWindow::setScrollBarsVisible(const bool visible)
+{
+    qDebug() << "setScrollBarsVisible: " << visible;
+    m_scrollBarsVisible = visible;
+
+    m_horizontalScrollBar->setVisible(visible);
```

Comment:
I wonder if `QAbstractScrollArea::setHorizontalScrollBarPolicy( Qt::ScrollBarAlwaysOff )` is better?

https://stackoverflow.com/questions/3383260/how-to-hide-scrollbar-in-qscrollarea

Reply:
I will investigate this, but according to some discussion on SO it seems this might result in the scroll bar object disappearing completely (i.e. not existing) which could create further issues. Let me take a look and circle back.